session/object.go

```diff
@@ -162,7 +162,7 @@ func (x *Object) UnmarshalJSON(data []byte) error {
 // expected to be calculated as a final stage of Object formation.
 //
 // See also VerifySignature.
-func (x *Object) Sign(signer neofscrypto.Signer) error {
+func (x *Object) Sign(signer user.Signer) error {
```

Comment:
Not needed.

Reply:
`x.sign` [uses it](https://github.com/nspcc-dev/neofs-sdk-go/blob/428-simpilfy-user-id-derivation-from-key/session/common.go#L158)